tests/test_random_variables/test_categorical.py

```diff
@@ -0,0 +1,107 @@
+"""Tests for the categorical random variable."""
```

Comment:
since you know pytest better than I do, please dont hesitate to criticise!

Reply:
Do I ? 😅 I have nothing to criticise